benchmark/collective/resnet/utils/img_tool.py

```diff
@@ -0,0 +1,134 @@
+""" tools for processing images
```

Comment:
please add license here. and mark that the file is mainly from models.

Reply:
Done.